docs/database/cosmosdb.md

```diff
@@ -440,5 +448,743 @@ It captures the number of active instances at any given time. Best practices dic
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Network Level Metrics
+
+### Metric: `db.client.cosmosdb.request.duration`
+
+This metric is [recommended][MetricRecommended].
+
+It captures the end to end Latency of Azure Cosmos DB Backend network call.
+
+this metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.35.0/specification/metrics/api.md#instrument-advisory-parameters)
+of `[ 0.001, 0.005, 0.01, 0.05, 0.1, 0.5, 1, 5, 10 ]`.
+
+<!-- semconv metric.db.client.cosmosdb.request.duration -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `db.client.cosmosdb.request.duration` | Histogram | `s` | End to end Latency of Azure Cosmos DB Backend network call | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`db.collection.name`](/docs/attributes-registry/db.md) | string | Cosmos DB container name. [1] | `public.users`; `customers` | `Conditionally Required` If available | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Please be specific whether it is CollectionName or RID. Also Comsos DB does not use term collection but Container for NoSQL Accounts. I assume you do this to be consistent with OpenTelemetry - but given that this is in the COsmos namespace shouldn't you use consistent terminology with Cosmos DB?

Reply:
Idea here is to keep things consistent with otel conventions. Description about the attribute would tell us what does it really mean, in terms of cosmosdb.